networking/multiple_networks/configuring-multi-network-policy.adoc

```diff
@@ -6,7 +6,15 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, you can configure a multi-network policy for a Single-Root I/O Virtualization (SR-IOV), MAC Virtual Local Area Network (MacVLAN), or OVN-Kubernetes additional networks. MacVLAN additional networks are fully supported. Other types of additional networks, such as IP Virtual Local Area Network (IPVLAN), are not supported. 
+As a cluster administrator, you can configure a multi-network policy for any of the following network types:
+
+* Single-Root I/O Virtualization (SR-IOV)
+* MAC Virtual Local Area Network (MacVLAN)
+* IP Virtual Local Area Network (IPVLAN)
+* Bond Container Network Interface (CNI) over SR-IOV
+* OVN-Kubernetes additional networks
+
+MacVLAN additional networks are fully supported.
```

Comment:
Do we know what this statement exactly means? It should be the same as `MAC Virtual Local Area Network (MacVLAN)`.

Reply:
I could not defend a position if a user claims an explanation, so it's okay to remove it from my side.

Reply:
Is there a concept of partial support and full support for the other network types?
It is only in this case I think this statement makes sense.

Reply:
There is only one note about the SR-IOV networks, that is stated right after this paragraph:
https://github.com/openshift/openshift-docs/pull/84671/files#diff-39bf1f67d4adfc408401fdf1ee603afb080600e2dfa028993efa249aad639d8fR21

Reply:
Removed "MacVLAN additional networks are fully supported." 
Thanks!